### PR TITLE
adding ssh keys to jwt token

### PR DESCRIPTION
**Pull request by alikyusupov**
Pull request to dev

### CODE REVIEW

File: middleware/isAuth.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
 const jwt = require("jsonwebtoken")
```
Comment (LarsMoran) [inline middleware/isAuth.js:1]: Always create an empty line at the end of all files. It's a github requirement

File: middleware/isAuth.js [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
-// const path = require("path")
-// const fs = require("fs")
+const path = require("path")
+const fs = require("fs")
 
 
 exports.isAuth = (req, res, next)=>{
-    // const token = fs.readFileSync(path.resolve("auth","private.key"),{encoding:"utf8"})
+    const token = fs.readFileSync(path.resolve("auth","jwtRS256.key.pub"),{encoding:"utf8"})
```
Comment (LarsMoran) [inline middleware/isAuth.js:7]: Move all functionality connected to JWT to a dedicated folder, e.d. /utils/jwt in the root of the project.

File: middleware/isAuth.js [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
     const clientToken = req.headers.authorization.split(" ")[1]
-    // console.log("token " , token)
-    // console.log("clientToken", clientToken)
-    jwt.verify(clientToken, "privateKey", (err, data)=>{
+    console.log("token " , token)
```
Comment (LarsMoran) [inline middleware/isAuth.js:9]: It's not a best practice to save console.log in commits. If you want to show any logs in development mode, use any log system ( can be created by yourself ). Find best practices of it

File: utils/JwtSignPromise.js [hunk 4]
```diff
@@ -1,10 +1,13 @@
...
+const fs = require("fs")
 
 exports.JwtSignPromise = (email, userID)=>{
+    const privateKey = fs.readFileSync(path.resolve("auth","jwtRS256.key"))
     return new Promise((resolve, reject)=>{
         jwt.sign({email, userID},
-            "privateKey",
-            {expiresIn:"1h"},
+            privateKey,
+            {algorithm:"RS256",expiresIn:"1h"},
```
Comment (LarsMoran) [inline utils/JwtSignPromise.js:10]: If you create jwt with so small life cycle, you need to create a refresh token, that will be stored in your DB. **Read best practices of it.** If you don't want to create refresh token functionality, make access token lifecycle about 18h-24h

File: middleware/isAuth.js (hunk 3 above)
Comment (alikyusupov) [reply middleware/isAuth.js:9]: Ok Got it

File: utils/JwtSignPromise.js (hunk 4 above)
Comment (alikyusupov) [reply utils/JwtSignPromise.js:10]: Which strategy is opt to production?

File: middleware/isAuth.js (hunk 2 above)
Comment (alikyusupov) [reply middleware/isAuth.js:7]: Ok Will fix it

**Pull request merged**